pkg/kubectl/cmd/explain.go

```diff
@@ -57,7 +57,6 @@ func NewCmdExplain(f cmdutil.Factory, out, cmdErr io.Writer) *cobra.Command {
 		},
 	}
 	cmd.Flags().Bool("recursive", false, "Print the fields of fields (Currently only 1 level deep)")
-	cmd.Flags().String("api-version", "", "The API version to use when talking to the server")
```

Comment:
I think you still need this one to deal with explaining resources for particular versions.  Is there another way to describe those?